homeassistant/components/somfy/cover.py

```diff
@@ -36,10 +38,12 @@ def get_covers():
 class SomfyCover(SomfyEntity, CoverDevice):
     """Representation of a Somfy cover device."""
 
-    def __init__(self, device, api):
+    def __init__(self, device, api, shadow):
```

Comment:
Can we rename this to something more clear than shadow? Maybe just keep optimistic?

Reply:
Done